src/common/ant.ts

```diff
@@ -540,6 +540,29 @@ export class AoANTWriteable extends AoANTReadable implements AoANTWrite {
     });
   }
 
+  /**
+   * @param logo @type {string} - Arweave transaction id of the logo we want to set
+   * @param options @type {WriteOptions} - additional options to add to the write interaction (optional)
+   * @returns {Promise<AoMessageResult>} The result of the interaction.
+   * @example
+   * ```ts
+   * ant.setLogo({ logo: "U7RXcpaVShG4u9nIcPVmm2FJSM5Gru9gQCIiRaIPV7f" });
+   * ```
+   */
+  async setLogo(
+    { logo }: { logo: string },
```

Comment:
nit: make this `txId`

Reply:
[cda5e1d](https://github.com/ar-io/ar-io-sdk/pull/268/commits/cda5e1d892d84a2160f72874833f518024aebf5d)